### PR TITLE
Bug 1732995, clarifying load balancer requirement

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1732995

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@gpei PTAL

**Comment by ahardin-rh** [comment]
@jianlinliu PTAL

**Comment by jianlinliu** [comment]
LGTM.

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -84,7 +84,8 @@ for platform containers and provide telemetry data to Red Hat.
 .Load balancers
 
 Before you install {product-title}, you must provision two layer-4 load
-balancers.
+balancers. The API needs one load balancer and the default Ingress controller
```
Comment (adellape) [inline modules/installation-network-user-infra.adoc]: s/needs/requires/g ?

Comment (adellape) [reply modules/installation-network-user-infra.adoc]: s/Ingress controller/Ingress Controller/

File: modules/installation-network-user-infra.adoc [hunk 2]
```diff
@@ -84,7 +84,8 @@ for platform containers and provide telemetry data to Red Hat.
...
+needs the second load balancer to provide Ingress to applications.
```
Comment (adellape) [inline modules/installation-network-user-infra.adoc]: s/Ingress/ingress/

Only this one, not the controller name.

**Review by adellape — COMMENTED** [review]
Minor suggestions.

**Pull request merged**